plugins/env/app/decorators/deploy_decorator.rb

```diff
@@ -3,6 +3,10 @@
 Deploy.class_eval do
   before_create :store_env_state
 
+  def retrieve_env_state
```

Comment:
still kinda fishy method, but overwriting env_state would also be weird ... at least it's in one place now ...